src/preact/base-element.js

```diff
@@ -212,7 +239,7 @@ export class PreactBaseElement extends AMP.BaseElement {
   unmount_() {
     this.mounted_ = false;
     if (this.container_) {
-      render(<></>, this.container_);
+      render(null, this.container_);
```

Comment:
Can you explain this change? It looks like it wont have any behavioral differences right? Do you expect a performance improvement?

Reply:
Yes. Turned out this is the recommended way to unmount in Preact. Mostly the same in behavior, but (1) no extra allocation, and (2) this is a true unmount - nothing else gets remounted as the result.